docs/ml-features.md

```diff
@@ -807,6 +809,36 @@ for more details on the API.
 </div>
 </div>
 
+## OneHotEncoderEstimator
+
+[One-hot encoding](http://en.wikipedia.org/wiki/One-hot) maps a column of label indices to a column of binary vectors, with at most a single one-value. This encoding allows algorithms which expect continuous features, such as Logistic Regression, to use categorical features.
```

Comment:
We should add a note that it can handle multiple columns (and returns a one-hot-encoded output vector column for _each_ input column, rather than merging into one output vector).

Also, what about describing the missing / invalid value handling in more detail?